judge/management/commands/runmoss.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import print_function
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from moss import MOSS
+
+from judge.models import *
+
+
+class Command(BaseCommand):
+    help = 'Checks for duplicate code using MOSS'
+
+    LANG_MAPPING = {
+        ('C++', MOSS_LANG_CC),
+        ('C', MOSS_LANG_C),
+        ('Java', MOSS_LANG_JAVA),
+        ('Python', MOSS_LANG_PYTHON),
+    }
+
+    def add_arguments(self, parser):
+        parser.add_argument('contest', help='the id of the contest')
+
+    def handle(self, *args, **options):
+        moss_api_key = settings.MOSS_API_KEY
+        if moss_api_key is None:
+            print('No MOSS API Key supplied')
+            return
+        contest = options['name']
+
+        for problem in Contest.objects.get(key=contest).problems.order_by('code'):
+            print('========== %s / %s ==========' % (problem.code, problem.name))
+            for dmoj_lang, moss_lang in lang_mapping:
+                print("%s: " % dmoj_lang, end=' ')
+                subs = Submission.objects.filter(contest__participation__virtual__in=(0,1), contest__participation__contest__key=contest,
```

Comment:
Can you seriously reduce the indentation here? This looks so... messed up. Also, put a space after `,`.